eth/main.cpp

```diff
@@ -340,6 +341,10 @@ int main(int argc, char** argv)
 	/// Wallet password stuff
 	string masterPassword;
 
+	// make sure data dir exists
+	if (!boost::filesystem::create_directory(getDataDir()))
+		BOOST_THROW_EXCEPTION(FileError());
```

Comment:
We already throw a lot of those uncaught exceptions around in the CLI from inside other functions that are called in main(). But yes I agree. A message is in order